webpack.config.js

```diff
@@ -220,7 +220,7 @@ const minimizer = [
 /** @type {webpack.Output} */
 const output = {
   filename: isDev || isTest ? 'bundle.js' : 'bundle.[hash].js',
-  path: path.join(__dirname, 'dist'),
+  path: path.join(__dirname, 'static-dist'),
```

Comment:
I would have let the generation in the dist directory.
You may move the files only when generating the cloudfront dist, but I don't think we need to change the actual build structure.